operator/src/main/java/oracle/kubernetes/weblogic/domain/model/Container.java

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2021, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.weblogic.domain.model;
+
+import java.util.Optional;
+
+import oracle.kubernetes.json.Description;
+import oracle.kubernetes.json.EnumClass;
+import oracle.kubernetes.operator.ImagePullPolicy;
+import oracle.kubernetes.operator.helpers.KubernetesUtils;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+public class Container {
+
+  public static final String INIT_CONTAINER_NAME_PREFIX = "operator-common-container";
+  public static final String DEFAULT_INIT_CONTAINER_COMMAND = "cp -R $COMMON_MOUNT_PATH/* $COMMON_TARGET_PATH";
+
+  /**
+   * The WDT resources container image.
+   */
+  @Description("The common mount container image name.")
+  private String image;
+
+  @Description(
+          "The image pull policy for the common mount container image. "
+                  + "Legal values are Always, Never, and IfNotPresent. "
+                  + "Defaults to Always if image ends in :latest; IfNotPresent, otherwise.")
+  @EnumClass(ImagePullPolicy.class)
+  private String imagePullPolicy;
+
+  @Description("The name for the operator created init container. Set to 'operator-common-container-${CONTAINER_NUM}'.")
+  private String name;
+
+  @Description("The init container command. Defaults to 'cp -R $COMMON_DIR/* $TARGET_DIR'.")
```

Comment:
How about:

The command for this init container. Defaults to 'cp -R $COMMON_DIR/* $TARGET_DIR'. This is an advanced setting for customizing the container command for copying files from the container image to the common mount emptyDir volume. Use the 'COMMON_DIR' environment variable to reference the value configured in 'commonMount.mountPath' (which  defaults to '/common'). Use 'TARGET_DIR' to refer to the temporary directory created by the Operator that resolves to the common mount's internal emptyDir volume.